src/um/winbase.rs

```diff
@@ -2784,13 +2784,13 @@ extern "system" {
         Context: PCONTEXT,
         FeatureMask: PDWORD64,
     ) -> BOOL;
-    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    #[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
     pub fn LocateXStateFeature(
         Context: PCONTEXT,
         FeatureId: DWORD,
         Length: PDWORD,
     ) -> PVOID;
-    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    #[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
```

Comment:
Same for these, they are based on specific architectures, not architecture width.